src/native/card.rs

```diff
@@ -559,7 +559,30 @@ where
             &self.style,
         );
     }
-}
+
+    fn overlay<'b>(
+            &'b mut self,
+            tree: &'b mut Tree,
+            layout: Layout<'_>,
+            renderer: &Renderer,
+        ) -> Option<core::overlay::Element<'b, Message, Renderer>> {
+            let mut children = vec![&mut self.head, &mut self.body];
+            if let Some(foot) = &mut self.foot {
+                children.push(foot);
+            }
+            let children = children
+                .into_iter()
+                .zip(&mut tree.children)
+                .zip(layout.children())
+                .filter_map(|((child, state), layout)| {
+                    let first_child = layout.children().next().unwrap();
```

Comment:
I think it should be possible to ignore the panic here by using a If Let Some(first_child) = layout.children().next().
you can make it return a None on an Else and then just filter out all the None's before Collect.